src/Illuminate/Foundation/Console/AboutCommand.php

```diff
@@ -163,6 +163,7 @@ protected function gatherApplicationInformation()
 
         $formatEnabledStatus = fn ($value) => $value ? '<fg=yellow;options=bold>ENABLED</>' : 'OFF';
         $formatCachedStatus = fn ($value) => $value ? '<fg=green;options=bold>CACHED</>' : '<fg=yellow;options=bold>NOT CACHED</>';
+        $formatStorageLinkedStatus = fn ($value) => $value ? '<fg=green;options=bold>LINKED</>' : '<fg=yellow;options=bold>NOT LINKED</>';
```

Comment:
$formatStorageLinkedStatus = fn ($value) => $value ? '<fg=green;options=bold>AVAILABLE</>' : '<fg=yellow;options=bold>NOT AVAILABLE</>';

Reply:
Or it can also be set to exists or not exists.

Reply:
@selcukcukur Hummm... the command that you'd run would be `artisan storage:link` which is why I went with Linked.  I don't have any strong preferences though.